### PR TITLE
feat(content): add a profile for Mathias

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
Add a profile for Mathias

### CODE REVIEW

**Comment by mahula** [comment]
In case these linting rule apply, I am out.

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ module.exports = {
   head: [
     ['meta', {name: 'viewport', content: 'width=device-width,initial-scale=1'}]
   ],
-  theme: 'vuepress-theme-book',
+  // theme: 'vuepress-theme-book',
+  theme: 'book',
```
Comment (Elweyn) [inline docs/.vuepress/config.js:9]: What the difference between ``theme: 'vuepress-theme-book'`` and ``theme: 'book',``?
I could not identify what this changed!

File: docs/people/mathias-lenz.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  - conception and test planning
+  - test automation
+  - agile testing
+  - manual testing
+  - exploratory testing
+- translations for OSS projects
+- contribution to open source gaming 3D multiplayer enginge
+  - porting from Java to C++ and patching
+  - setting up and maintaining build systems for Unix operating systems
+  - patching
```
Comment (Elweyn) [inline docs/people/mathias-lenz.md:53]: Why is here no capital letter at the front?

File: docs/people/mathias-lenz.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+- [C++](https://en.wikipedia.org/wiki/C%2B%2B)
+- [HTML](https://en.wikipedia.org/wiki/HTML)
+- [CSS](https://en.wikipedia.org/wiki/CSS)
+- [LaTex](https://www.latex-project.org/)
+
+
+## Further commitments and interests
+
+Besides computer technology there are commitments and interests in
+- deleicious and healthy food and cooking
```
Comment (Elweyn) [inline docs/people/mathias-lenz.md:108]: Capital letter and free line between the sentence and the bulletlist

**Review by Elweyn — COMMENTED** [review]
I like the structure you used, just some small questions / suggestions

File: docs/people/mathias-lenz.md (hunk 2 above)
Comment (mahula) [reply docs/people/mathias-lenz.md]: Fixed with https://github.com/IT4Change/IT4C.dev/pull/36/commits/d557815ae8d010477d8133a42705536f1b646b6c.

File: docs/people/mathias-lenz.md (hunk 3 above)
Comment (mahula) [reply docs/people/mathias-lenz.md]: Fixed with bbc4734.

File: docs/.vuepress/config.js (hunk 1 above)
Comment (mahula) [reply docs/.vuepress/config.js]: Thank you for the hint.
The theme is set back to `vuepress-theme-book`.

File: docs/people/mathias-lenz.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+- Translations for OSS projects
+- Contribution to open source gaming 3D multiplayer enginge
+  - Porting from Java to C++ and patching
+  - Setting up and maintaining build systems for Unix operating systems
+  - Patching
+
+
+## Approaches & tools
+
+### Testing approaches
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:58]:
```suggestion
### Testing Approaches
```

File: docs/people/mathias-lenz.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+- [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development)
+- [Behavior-driven development/testing](https://www.bddtesting.com/)
+- Testing by concept and plan
+- Exploratoy testing (based on intuition and experience)
+
+
+### Test automation
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:66]:
```suggestion
### Test Automation
```

File: docs/people/mathias-lenz.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+- [Cypress](https://www.cypress.io/)
+- [Selenium](https://www.selenium.dev/)
+- [Appium](https://appium.io/)
+- [RSpec](https://rspec.info/)
+- [Insomnia](https://insomnia.rest/)
+- [Cucumber](https://cucumber.io/) and [Gherkin](https://cucumber.io/docs/gherkin/)
+- [Katalon Studio](https://katalon.com/)
+
+
+### Test management & reporting
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:79]:
```suggestion
### Test Management & Reporting

```

File: docs/people/mathias-lenz.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+- [Trac](https://trac.edgewall.org/)
+- [TestRail](https://www.gurock.com/testrail/)
+- [Xray](https://www.getxray.app/)
+- [Github Issues](https://docs.github.com/en/issues)
+- [GitLab Test Cases](https://docs.gitlab.com/ee/ci/test_cases/)
+- [Jira bug tracking](https://www.atlassian.com/software/jira/features/bug-tracking)
+
+
+### Frontend development
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:88]:
```suggestion
### Frontend Development
```

File: docs/people/mathias-lenz.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+
+- [Vue](https://vuejs.org/)
+- [Quasar](https://quasar.dev/)
+
+
+### Programming & markup languages
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:94]:
```suggestion
### Programming & Markup Languages
```

File: docs/people/mathias-lenz.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+- [Javascript](https://www.javascript.com/)
+- [Python](https://www.python.org/)
+- [Ruby](https://www.ruby-lang.org/)
+- [C++](https://en.wikipedia.org/wiki/C%2B%2B)
+- [HTML](https://en.wikipedia.org/wiki/HTML)
+- [CSS](https://en.wikipedia.org/wiki/CSS)
+- [LaTex](https://www.latex-project.org/)
+
+
+## Further commitments and interests
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:105]:
```suggestion
## Further Commitments and Interests
```

File: docs/people/mathias-lenz.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+- Deleicious and healthy food and cooking
+- Msic - fine patterns and lyrics
+- Human connections
+- Literature
+- Time exchange
+- Connecting the dots
+
+
+## Spoken languages
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:117]:
```suggestion
## Spoken Languages
```

File: docs/people/mathias-lenz.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
+- [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development)
+- [Behavior-driven development/testing](https://www.bddtesting.com/)
+- Testing by concept and plan
+- Exploratoy testing (based on intuition and experience)
+
+
+### Test automation
+
+- [Probot Framework](https://robotframework.org/)
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:68]:
```suggestion
- [Robot Framework](https://robotframework.org/)
```

File: docs/people/mathias-lenz.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+- [HTML](https://en.wikipedia.org/wiki/HTML)
+- [CSS](https://en.wikipedia.org/wiki/CSS)
+- [LaTex](https://www.latex-project.org/)
+
+
+## Further commitments and interests
+
+Besides computer technology there are commitments and interests in
+
+- Deleicious and healthy food and cooking
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:109]:
```suggestion
- Delicious and healthy food and cooking
```

File: docs/people/mathias-lenz.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+- Msic - fine patterns and lyrics
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:110]: May be?

```suggestion
- Misc - fine patterns and lyrics
```

File: docs/people/mathias-lenz.md [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+
+## Spoken languages
+
+- German (native speaker)
+- English (business fluent)
+
+
+### Profiles
+
+[Github](https://github.com/mahula)
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:125]:
```suggestion
[GitHub](https://github.com/mahula)
```

File: docs/people/mathias-lenz.md [hunk 15]
```diff
@@ -0,0 +1,125 @@
+# Mathias 
+
+![Mathias](~@images/portrait/ml-portrait.jpg)
+
+Rooted in DIY crafting and interest in language, patterns, and human connections.
+In school years first self taught steps where made towards computer systems and programming.
+While studying computational linguistics Mathias branched out into different areas of the computer technology realm.<br />
+His motivation is to spread the knowledge and hands-on possibilities of contemporary tools, reduce scepticism about them, and empower anybody interested to get hold of them and use them to their own benefits.
+And in particular in software development´s facette of quality assurance he found his profession.
```
[the remark this answers is not in the document]
Comment (mahula) [reply docs/people/mathias-lenz.md:9]: This is due to a linting rule ...

File: docs/people/mathias-lenz.md [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+- Connecting the dots
+
+
+## Spoken languages
+
+- German (native speaker)
+- English (business fluent)
+
+
+### Profiles
```
Comment (Tirokk) [inline docs/people/mathias-lenz.md:123]:
```suggestion
## Profile
```

File: docs/people/mathias-lenz.md [hunk 17]
```diff
@@ -0,0 +1,125 @@
+# Mathias 
+
+![Mathias](~@images/portrait/ml-portrait.jpg)
+
+Rooted in DIY crafting and interest in language, patterns, and human connections.
+In school years first self taught steps where made towards computer systems and programming.
+While studying computational linguistics Mathias branched out into different areas of the computer technology realm.<br />
+His motivation is to spread the knowledge and hands-on possibilities of contemporary tools, reduce scepticism about them, and empower anybody interested to get hold of them and use them to their own benefits.
```
[the remark this answers is not in the document]
Comment (mahula) [reply docs/people/mathias-lenz.md:8]: I'll stick with BE.

File: docs/people/mathias-lenz.md [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+  - Consulting on test strategies and tools
+  - Setup of test environments
+  - Agile testing
+  - Manual testing
+  - Exploratory testing
+- DevOps
+- Frontend development
+- Design-Eye
+
+And with a hands-on mentality, he can also contribute to other areas and familiarise himself with new approaches and tools to get an idea or project to work.
```
[the remark this answers is not in the document]
Comment (mahula) [reply docs/people/mathias-lenz.md:29]: I'll stick with BE.

File: docs/people/mathias-lenz.md (hunk 11 above)
Comment (mahula) [reply docs/people/mathias-lenz.md:68]: Thank you.

File: docs/people/mathias-lenz.md [hunk 19]
```diff
@@ -0,0 +1,125 @@
+# Mathias 
+
+![Mathias](~@images/portrait/ml-portrait.jpg)
+
+Rooted in DIY crafting and interest in language, patterns, and human connections.
+In school years first self taught steps where made towards computer systems and programming.
+While studying computational linguistics Mathias branched out into different areas of the computer technology realm.<br />
```
[the remark this answers is not in the document]
Comment (mahula) [reply docs/people/mathias-lenz.md]: Thank you.
Fixed with https://github.com/IT4Change/IT4C.dev/pull/36/commits/24804870246a8c9740a6ac2a8bfca3c7c0115869

File: docs/people/mathias-lenz.md (hunk 13 above)
Comment (mahula) [reply docs/people/mathias-lenz.md]: Thank you.
Fixed with https://github.com/IT4Change/IT4C.dev/pull/36/commits/24804870246a8c9740a6ac2a8bfca3c7c0115869

**Review by Tirokk — APPROVED** [review]
YEAH!

**Comment by ulfgebhardt** [comment]
linting rules can be disabled if needed `<!-- textlint-disable write-good -->` and reenabled: `<!-- textlint-enable write-good -->`

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**